applications/madini_tracechain.md

```diff
@@ -129,37 +170,75 @@ Project Details
 
 We expect the teams to already have a solid idea about the project's expected final state. Therefore, we ask the teams to submit (where relevant):
 
-    Mockups/designs of any UI components
```

Comment:
![madiniwallet-1](https://user-images.githubusercontent.com/8124128/140720375-851830cb-dd1f-416d-91ef-404138ff2766.jpg)

**Madini Wallet**

Madini Wallet is a light mobile wallet app based on VeChainThor. We have forked  VeChain github original code to buid Madini Wallet specific features and completely change the design and usability in Java. We aim to provide users with a powerful, secure, simple, fully functional portal of the Madini VeChainThor blockchain.
Main functions include: MDN, VET, VTHO and other Crypto assets management, Balance, Exchange using Nomics API, Graph, Blockchain Inspector to check your transactions status, wallet observation, reward claim, etc.
Robust Security Protections Madini VeChainThor Wallet bas been undergoing a series of security testing with trusted third party security firms, including functional test, source code review, smart contract audit, session management audit, input security audit, etc.
Smart Crypto Asset Management Madini wallet provide a secure and effective environment for cryptocurrency transactions. Users can effectively perform MDN and VET token transfer and observe the balance of your crypto assets.

Technical Details of Madini Wallet
•	Programming language used:Android JAVA



![tracechainweb](https://user-images.githubusercontent.com/8124128/140721115-6bf76da5-9709-41c1-88db-51fc0201037e.jpg)
**Madini TraceChain**

Madini TraceChain Technical and Architecture

TraceChain interfaces with both block-chain and a server with a database.Madini TraceChain relies on the Vechain blockchain a distributed block-chain service.Verified data is broadcasted to the blockchain and stored as hex data. Madini TraceChain relies on the Vechain blockchain a distributed block-chain service.Verified data is broadcasted to the blockchain and stored as hex data also stores the raw data to the SQLite 

Madini TraceChain Technical Details:

•Python 3.8 (Flask)
•VPS / Server.
•Sqlite Database.
•VET Wallet address with VTHO.
•VIP 190